osquery/tables/networking/darwin/wifi_utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreWLAN/CoreWLAN.h>
+#include <iomanip>
+// SSIDs have no character set associated with them
```

Comment:
These doc blocks are incorrect. Take a look at the other headers for reference (in ./include/osquery).